app/models/tasuku/taskables/question/answer.rb

```diff
@@ -15,7 +15,13 @@ def question
         votes.first.option.question if votes.any?
       end
 
-      request is: :question
+      def request
+        if question.quizzes.present?
+          question.quizzes.first
```

Comment:
Current design is that question can belong to multiple quizes and be stand alone question at the same time. But such scenario breaks submission process and proper statistics calculation. This has to be reworked somewhen, but the fix is work around for now